source/reference/configure-shell-settings-global.txt

```diff
@@ -0,0 +1,104 @@
+.. _configure-settings-global:
+
+===========================================================
+Configure :binary:`~bin.mongosh` Using a Configuration File
```

Comment:
[Note for review]

This page is entirely new, and contains the bulk of what needs to be reviewed in this PR.